Makefile

```diff
@@ -56,6 +56,9 @@ render-chart:
 	helm template ks-ctl charts/ks-devops  -n kubesphere-devops-system \
 		--set serviceAccount.create=true --create-namespace \
 		--set image.pullPolicy=Always \
+#		--set jenkins.Agent.Builder.Registry=ghcr.io/linuxsuren \
+#		--set jenkins.Agent.Builder.Maven.Tag=test-podman \
```

Comment:
test-podman -> test.

Reply:
See also https://github.com/LinuxSuRen/devops-agent/pkgs/container/builder-maven

![image](https://user-images.githubusercontent.com/1450685/129508438-da1557bf-36bd-4b9d-ad51-54d3e7c69d78.png)

Reply:
By the way, the line 59 and 60 is just an example for user if they want to change the image name and tag.

Reply:
I found that we will append `-podman` to the maven tag by default if we choose podman as container runtime.

https://github.com/kubesphere/ks-devops/blob/6024df1d54657f7a3454ba1db83709d2b883fb96/charts/ks-devops/charts/jenkins/templates/_helpers.tpl#L50-L54

https://github.com/kubesphere/ks-devops/blob/6024df1d54657f7a3454ba1db83709d2b883fb96/charts/ks-devops/charts/jenkins/templates/jenkins-casc-config.yml#L108-L111

And I also found [there](https://github.com/LinuxSuRen/devops-agent/pkgs/container/builder-maven) was a image with `test-podman-podman` tag. But I think it is so weird.

Reply:
That's not a problem. As I mentioned, line 59 and 60 are just the example usage.